zulipterminal/ui_tools/boxes.py

```diff
@@ -430,7 +439,9 @@ def main_view(self) -> List[Any]:
         # Build parts together and return
         parts = [
             (recipient_header, recipient_header is not None),
-            (content_header, any_differences),
+            (content_header, any_differences or
```

Comment:
Can the condition be `is_content_header` (or `add_content_header`), or even just `content_header`?